.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+.DS_Store
```

Comment:
`.DS_Store` is a temp file that you want to ignore in your projects and not commit